### PR TITLE
added post and get api for popularProductEnquiry

### PR DESCRIPTION
**Pull request by ChaitanyaV98**
- added post and get api calls for popularProductEnquiry.

### CODE REVIEW

File: app/controllers/popularProductEnquiry.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    try {
+      let getAllPopularProductEnquiry = await PopularProductEnquiry.findAll({
+        where: {},
+      });
+      return getAllPopularProductEnquiry;
+    } catch (error) {
+      consumeError(error);
+    }
+  },
+  async create(req, res) {
```
Comment (ShubhamJoshi-1994) [inline app/controllers/popularProductEnquiry.js:16]: can we implement Validations here? @ChaitanyaV98

File: app/controllers/popularProductEnquiry.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    try {
+      const popularProductEnquiry = await PopularProductEnquiry.create({
+        name: req.body.name,
+        product_name: req.body.product_name,
```
Comment (ShubhamJoshi-1994) [inline app/controllers/popularProductEnquiry.js:20]: what if we are getting product_uuid in payload?

Comment (ShubhamJoshi-1994) [reply app/controllers/popularProductEnquiry.js:20]: where are you saving the user_uuid?

Comment (ShubhamJoshi-1994) [reply app/controllers/popularProductEnquiry.js:20]: where are we passing the status? @ChaitanyaV98

**Review by ShubhamJoshi-1994 — CHANGES_REQUESTED** [review]

File: app/controllers/popularProductEnquiry.js (hunk 1 above)
Comment (ChaitanyaV98) [reply app/controllers/popularProductEnquiry.js:16]: we have already added validations separately in requests folder! @ShubhamJoshi-1994

File: app/controllers/popularProductEnquiry.js (hunk 2 above)
Comment (ChaitanyaV98) [reply app/controllers/popularProductEnquiry.js:20]: 1. @shubham all the popular products that we are showing are hard coded and we dont have uuid of the product @ShubhamJoshi-1994 .

Comment (ChaitanyaV98) [reply app/controllers/popularProductEnquiry.js:20]: I have changed the code @ShubhamJoshi-1994, now we are able to add user_uuid and status as well.

**Pull request merged**